src/encoded/audit/experiment.py

```diff
@@ -4283,6 +4283,36 @@ def check_experiment_atac_encode4_qc_standards(experiment, files_structure):
                     yield AuditFailure('mild to moderate bottlenecking', pbc2_detail, level='WARNING')
 
 
+def audit_analysis_files(value, system, excluded_types):
+    if 'analysis_objects' not in value:
+        return
+    experiment_files = (
+        f['@id']
+        for f in (
+            value.get('original_files', [])
+            + value.get('contributing_files', [])
+        )
+    )
+    detail_list = []
+    for analysis in value['analysis_objects']:
+        for f in analysis.get('files', []):
+            if f not in experiment_files:
+                detail_list.append(
+                    'Analysis {} has a file {} which is not belong to this '
+                    'experiment {}.'.format(
```

Comment:
I think it should be "{} which does not belong" instead of "{} which is not belong"